### PR TITLE
Update to 1.0.0

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

Update to 1.0.0

## Test Plan

Manual test

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,299 +1,364 @@
...
 
 const listDocuments = async () => {
-  console.log(chalk.greenBright('Running List Documents API'));
+    console.log(chalk.greenBright('Running List Documents API'));
+
+    const response = await databases.listDocuments(
+        databaseId,
+        collectionId,
+        [
+            Query.equal('release_year', 1920),
```
Comment (stnguyen90) [inline src/app.js:132]: When I ran this, I got an error saying the index on `release_year` was not found:

```
AppwriteException [Error]: Index not found: release_year
    at Client.call (/workspace/playground-for-node/node_modules/node-appwrite/lib/client.js:171:31)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Databases.listDocuments (/workspace/playground-for-node/node_modules/node-appwrite/lib/services/databases.js:1004:16)
    at async listDocuments (/workspace/playground-for-node/src/app.js:128:22)
    at async runAllTasks (/workspace/playground-for-node/src/app.js:331:5) {
  code: 400,
  type: 'general_argument_invalid',
  response: {
    message: 'Index not found: release_year',
    code: 400,
    type: 'general_argument_invalid',
    version: '1.0.1'
  }
}
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: src/app.js [hunk 2]
```diff
@@ -1,299 +1,364 @@
...
+        'release_year',
+        false,
+        0, 5000,
+        1970,
+        false
+    );
+    console.log(yearAttributeResponse);
+
+    console.log("Waiting a little to ensure attributes are created ...");
+    await new Promise((resolve) => setTimeout(resolve, 2000));
```
Comment (stnguyen90) [inline src/app.js:76]: Add code to create index.

```suggestion
    await new Promise((resolve) => setTimeout(resolve, 2000));

    const yearIndexResponse = await databases.createIndex(
        databaseId,
        collectionId,
        'key_release_year_asc',
        'key',
        ['release_year'],
        ['ASC'],
    );
    console.log(yearIndexResponse);

    console.log("Waiting a little to ensure index is created ...");
    await new Promise((resolve) => setTimeout(resolve, 2000));
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Just missing index on release_year. Otherwise, LGTM!

File: src/app.js [hunk 3]
```diff
@@ -1,299 +1,377 @@
...
 
-  await addDocument();
-  await listDocuments();
+    await addDocument();
+    await listDocuments();
 
-  await deleteDocument();
-  await deleteCollection();
+    await deleteDocument();
+    await deleteCollection();
```
Comment (stnguyen90) [inline src/app.js:347]: should we add `deleteDatabase()` too?

Comment (abnegate) [reply src/app.js:514]: Added plus some more 👍

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**